front/app/containers/Admin/users/ChangeSeatModal.tsx

```diff
@@ -142,7 +142,7 @@ const ChangeSeatModal = ({
                 }}
               />
             </Text>
-            {!isChangingCollaboratorToNormalUser && (
+            {!isChangingToNormalUser && (
```

Comment:
We also not need to see the seat info if we're setting admin > normal user. The previous check only filtered it out for collaborator > normal user.